webapp/cas-server-webapp-init-tomcat/src/main/java/org/apereo/cas/config/CasEmbeddedContainerTomcatConfiguration.java

```diff
@@ -47,4 +51,11 @@ public ServletWebServerFactoryCustomizer casTomcatEmbeddedServletContainerCustom
         final CasConfigurationProperties casProperties) {
         return new CasTomcatServletWebServerFactoryCustomizer(serverProperties, casProperties);
     }
+
+    @ConditionalOnMissingBean(name = "casWebApplicationReadyListener")
+    @Bean
+    public CasWebApplicationReadyListener casWebApplicationReadyListener() {
+        return new CasWebApplicationReady();
```

Comment:
Moving this here removes the ready prompt from other WebApps. You might need to recreate the same bean for jertty and undertow in their own config.

Reply:
Oh, I mistakenly thought the main class was in the tomcat module. I made a configuration class in the cas-server-webapp-init module so jetty, tomcat and undertow can share it.